lib/widgets/check_out_form.dart

```diff
@@ -98,34 +99,70 @@ class _CheckoutFormState extends State<CheckoutForm> {
-          child: SizedBox(
-            child: Container(
-              padding: EdgeInsets.symmetric(
-                vertical: responsiveHeight(35.0),
-              ),
-              child: SingleChildScrollView(
-                child: Column(
-                  children: [
-                    Text(
-                      checkoutBike.tag,
-                      style: TextStyle(
-                        color: Colors.white,
-                        fontSize: responsiveWidth(18.0),
-                        fontWeight: FontWeight.w500,
-                      ),
+          child: SingleChildScrollView(
+            child: Column(
+              children: [
+                Padding(
+                  padding: EdgeInsets.only(top: responsiveHeight(20.0)),
+                  child: Text(
+                    '${checkoutBike.bikeName}',
+                    style: TextStyle(
+                      color: Colors.white,
+                      fontSize: responsiveWidth(25.0),
+                      fontWeight: FontWeight.w500,
                     ),
-                    SizedBox(height: responsiveHeight(20.0)),
-                    checkoutBike.photoUrl == null
-                        ? Icon(Icons.image_outlined,
-                            size: responsiveWidth(100.0))
-                        : Image.network(checkoutBike.photoUrl.toString()),
-                    SizedBox(height: responsiveHeight(20.0)),
-                    displayRating(checkoutBike),
-                    SizedBox(height: responsiveHeight(20.0)),
-                    checkoutButton(context, checkoutBike),
-                  ],
+                  ),
+                ),
+                SizedBox(height: responsiveHeight(20.0)),
+                Image.network(
+                  checkoutBike.photoUrl.toString(),
+                  loadingBuilder: (BuildContext context, Widget child,
+                      ImageChunkEvent loadingProgress) {
+                    if (loadingProgress == null) return child;
+                    return Center(
+                      child: CircularProgressIndicator(
+                        value: loadingProgress.expectedTotalBytes != null
+                            ? loadingProgress.cumulativeBytesLoaded /
+                                loadingProgress.expectedTotalBytes
+                            : null,
+                      ),
+                    );
+                  },
+                ),
+                SizedBox(height: responsiveHeight(15.0)),
+                Text(
+                  'Is this bike missing?',
+                  style: TextStyle(
+                    color: Colors.white,
+                    fontSize: responsiveWidth(15.0),
+                    fontWeight: FontWeight.w500,
+                  ),
+                ),
+                missingButton(context, checkoutBike),
+                SizedBox(height: responsiveHeight(15.0)),
+                Text(
+                  'Bike Type: ${checkoutBike.tag}',
```

Comment:
We need to figure out something regarding this line, currently the object is being stored as a string.  One simple but ugly fix would be to just make some function that converts it to a better looking string atleast.  If (checkoutBike.tag == "BikeType.MOUNTAIN") checkoutBike.tag = "Mountain Bike" or something like that